tests/cupy_tests/math_tests/test_window.py

```diff
@@ -14,3 +14,17 @@ class TestWindow(unittest.TestCase):
     @testing.numpy_cupy_allclose(atol=1e-5)
     def test_window(self, xp):
         return getattr(xp, self.name)(self.m)
+
+
+@testing.parameterize(
+    *testing.product({
+        'm': [0, 1, -1, 1024],
+        'beta': [0, 5, 6, 8.6],
+        'name': ['kaiser'],
+    })
+)
+class TestKaiser(unittest.TestCase):
+
+    @testing.numpy_cupy_allclose(atol=1e-5)
+    def test_kaiser(self, xp):
```

Comment:
Please, check special cases such as `m==0` or `m==-1` in dedicated tests

Reply:
I have added a new Test Class to specifically check for boundary conditions and changed the existing one to try values which are more dispersed.